packages/manager/apps/container/src/container/legacy/server-sidebar/order/shop-config/dedicated.ts

```diff
@@ -25,15 +14,15 @@ const dedicatedShopConfig = (
   features['public-cloud'] && ORDER_URLS[region].publicCloudProjectOrder
     ? {
         label: 'order_item_cloud_project',
-        icon: OVHFontPublicCloud,
+        icon: SvgIconWrapper({ name: 'ProjectCloud' }),
```

Comment:
**nitpick:** Maybe you can export the Icon names from `ovh-product-icons` similar to `OdsIconName` and use it instead of hard-coding the product names here